source/codegen/metadata/niswitch/enums.py

```diff
@@ -1,6 +1,4 @@
-# -*- coding: utf-8 -*-
-# This file is generated from NI-SWITCH API metadata version 19.6.0d7
```

Comment:
Is the metadata version now ignored? Seems a comment like this would still be helpful. But that's probably more a question for the generator.

Reply:
I'm not sure what I was looking at, but this looks correct again. So nevermind?